### DISCUSSION TITLE
Change size/spacing of objects

### DISCUSSION THREAD

**Discussion by DANgerous25** [matt8707/ha-fusion#568] [Ideas]
Thank you for such a great project! I have successfully got an Amazon Fire 8 HD working with the Fusion dashboard working with Fully Kiosk. It's a very nice solution. My only trouble is that I have too many UI elements to fit on one screen (and I want them all on a single screen).

Is it, or can it be possible, to change the sizing of objects, perhaps change the font size? Also have some spacing objects? I have tried using sections with nothing in them for spacing, but the space is just too big.

My problem is that the FireHD screen 8 is quite small, so I am trying to make the best of the available space ;-).

Thanks again for great work! (I've just made a donation btw)

Dan

**Comment by OlofBailey** [comment]
I have a similar issue. The resolution of the wall mounted tablets I have cuts off the right side of a four column layout..
